packages/web-pkg/src/composables/appDefaults/useAppConfig.ts

```diff
@@ -13,9 +12,8 @@ export interface AppConfigResult {
 }
 
 export function useAppConfig(options: AppConfigOptions): AppConfigResult {
-  const applicationMetaResult = useAppMeta(options)
   const applicationConfig = computed(
-    () => unref(applicationMetaResult.applicationMeta).config || {}
+    () => options.appsStore.externalAppConfig[options.applicationId] || {}
```

Comment:
Is this something else or just removing an indirection?

Reply:
As discussed, config was removed from the app meta.